packages/vaults/src/transaction/policy/policy_update.ts

```diff
@@ -9,13 +9,28 @@ import {TransactionRequest} from "../transaction_request";
 import {RequestMapperAbstract} from "../request_mapper";
 
 
+/**
+ * Interface for a transaction that updates an existing policy.
+ * This transaction can only be requested or approved only by users with the admin role.
+ * This transaction can be executed in a batch.
```

Comment:
This transaction can only be requested or approved by users with the admin role.
This transaction can be executed in a batch, which means that all transactions marked with a single batch_id are executed or rejected together.